.github/workflows/ci.yml

```diff
@@ -725,6 +725,12 @@ jobs:
           cd tee-worker/docker
           docker compose -f litentry-parachain.build.yml build
 
+      - name: Update metadata and generate types
+        timeout-minutes: 10
+        run: |
+          cd tee-worker/docker
+          docker compose -f docker-compose.yml -f lit-ts-api-package-build.yml up --no-build --exit-code-from lit-ts-api-package-build lit-ts-api-package-build
+
```

Comment:
This step will be run in each matrix branch :(

Reply:
Ok, change it to upload client-api to artifacts and then download and use it.